package.json

```diff
@@ -66,21 +70,32 @@
     "@angular/platform-browser": "15.2.2",
     "@angular/platform-browser-dynamic": "15.2.2",
     "@angular/router": "15.2.2",
-    "@cds/core": "6.9.2",
+    "@cds/core": "file:dist/@cds/core",
```

Comment:
Another problem with this is that the `cds-icon` imports work in this project, but the icon component is not included in a published package. I think this needs to be restructured so that there is not a `./dist/@cds/core` folder. It's not actually a dist folder anyway because it is not published. Instead, I think the styles/tokens should be in `@clr/ui` and the icon component should be in `@clr/angular`.

Reply:
It's still one of the WIP parts. And your feedback is very valuable.
The idea is to either have a separate package (but not named `@cds/core`, maybe `@clr/core` or something else), or, as you say, move tokens to `@clr/ui` and the rest to `@clr/angular` (or a smaller core package).
The problem with moving them to the existing packages is that many of the consumable artefacts should still be built before we're able to import and use them.
As a next step we can explore the merging option, and if unsuccessful, fall back to a `@clr/core` or similar package.